supplementary_style_guide/glossary_terms_conventions/product_conventions/jboss-eap.adoc

```diff
@@ -126,32 +1086,191 @@
 ==== standalone server (noun)
-*Description*: This is the appropriate way to refer to the standalone operating mode. For example, when running JBoss EAP as a standalone server.
+*Description*: Use "standalone server" to refer to the standalone operating mode of JBoss EAP server. For example, when running JBoss EAP as a standalone server.
 
 *Use it*: yes
 
 *Incorrect forms*: standalone mode
 
 *See also*: xref:standalone-mode[standalone mode]
 
+// ***********************
+// Terms starting with 'T'
+// ***********************
+
+[discrete]
+[[transactions]]
+==== transactions (noun)
+*Description*: The "transactions" subsystem is used to configure options in the Transaction Manager. Write in lowercase. See xref:transactions-heading[Transactions] entry for the correct usage in headings.
+
+*Use it*: yes
+
+*Incorrect forms*: Transactions
+
+*See also*: xref:transactions-heading[Transactions]
+
+[discrete]
+[[transactions-heading]]
+==== Transactions (noun)
+*Description*: Use "Transactions" when referring to the xref:transactions[transactions] subsystem in titles and headings. Capitalize the word. See xref:transactions[transactions] entry for the correct usage in general text.
+
+*Use it*: yes
+
+*Incorrect forms*: transactions
+
+*See also*: xref:transactions[transactions]
+
+[discrete]
+[[truststore]]
+==== truststore (noun)
+*Description*: A "truststore" is a repository of trusted security certificates. Write in lowercase as one word. This is in contrast to a "keystore", which stores private and self-certified certificates.
+
+*Use it*: yes
+
+*Incorrect forms*: trust store
+
+*See also*: xref:keystore[keystore]
+
+// ***********************
+// Terms starting with 'U'
+// ***********************
+
+[discrete]
+[[undertow]]
+==== undertow (noun)
+*Description*: The "undertow" subsystem is used to configure the JBoss EAP web server and servlet container settings. Write in lowercase. See xref:undertow-heading[Undertow] entry for the correct usage in headings.
+
+*Use it*: yes
+
+*Incorrect forms*: Undertow
+
+*See also*: xref:undertow-heading[Undertow]
+
+[discrete]
+[[undertow-heading]]
+==== Undertow (noun)
+*Description*: Use "Undertow" when referring to the xref:undertow[undertow] subsystem in titles and headings. Capitalize the word. See xref:undertow[undertow] entry for the correct usage in general text.
+
+*Use it*: yes
+
+*Incorrect forms*: undertow
+
+*See also*: xref:undertow[undertow]
+
+// ***********************
+// Terms starting with 'W'
+// ***********************
+
+[discrete]
+[[webhttp-undertow]]
+==== WebHTTP - Undertow (noun)
+*Description*: Use "WebHTTP - Undertow" when describing the undertow subsystem in the management console. Write as two capitalized words separated by two spaces and a hyphen. Ensure that "HTTP" is also in uppercase.
+
+*Use it*: yes
+
+*Incorrect forms*: Any form omitting the spaces, hyphen or correct capitalization
+
+*See also*: xref:undertow[undertow], xref:undertow-heading[Undertow]
```

Comment:
Should the undertow entries also list "WebHTTP - Undertow" in the "See also" field?

Reply:
Fixed